src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,21 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+
+    double foodCoastCoefficient = 0.8;
```

Comment:
becuase variables in interfaces are final they should follow the correct format.

`double FOOD_COST_COEFFICIENT = 0.8;`